lib/module.js

```diff
@@ -144,52 +182,24 @@ Module._findPath = function(request, paths, isMain) {
     return Module._pathCache[cacheKey];
   }
 
-  var exts;
+  const exts = Object.keys(Module._extensions);
   const trailingSlash = request.length > 0 &&
                         request.charCodeAt(request.length - 1) === 47/*/*/;
+  const isAbsolute = !trailingSlash;
 
   // For each path
   for (var i = 0; i < paths.length; i++) {
     // Don't search further if path doesn't exist
-    const curPath = paths[i];
+    var curPath = paths[i];
```

Comment:
@fengmk2 I was tracing this particular function (`Module._findPath()`) and this particular `const` usage did not result in aborted or disabled optimizations for the entire containing function.